feg/radius/src/modules/eap/authstate/manager.memory_test.go

```diff
@@ -43,7 +43,7 @@ func performSignleReadWriteDeleteReadTest(t *testing.T, manager Manager, authReq
 	// Arrange (randomize state)
 	correlationID := rand.Intn(9999999)
 	eapType := packet.EAPTypeAKA
-	protocolState := string(rand.Intn(999999))
+	protocolState := fmt.Sprint(rand.Intn(999999))
```

Comment:
Sorry, I am not a go expert - what is happening here?

Reply:
The original code converts the random integer into a rune, which is a single unicode character, while the current state converts it into a string of integers. Note that I have just changed the function I've used to convert it to a string of integers, but the effect is the same.

Without this change, I get the following error when I run the tests:
```
modules/eap/authstate/manager.memory_test.go:46:19: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
```
and the two tests where this function is used fail:
```
WARN[0000] FAIL fbc/cwf/radius/modules/eap/authstate [build failed] 
bad output from test2json: FAIL fbc/cwf/radius/modules/eap/authstate [build failed]
```

While with the change, the two tests pass. It's hard to know for certain what the intention of the original author was, but for the tests to pass any string will do. I.e. if I change it to `protocolState := ""`, the tests works.